### ISSUE TITLE
A11y_AzuredataStudio_Clustersettings_ScreenReader:Voice over does not announce expand and collapse state for "docker settings" button.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7940]
GitHubTags: #SQL Azure Data Studio;#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#WCAG4.1.2;#DesktopApp;#Voice Over;#Mac;
Environment Details:
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: MacOS Mojave 10.14.6
**Additional Details:**
Voice over
**Steps to Reproduce :**
1. Install /Launch the application Azure data studio.
2. Navigate to connections and click on more action button.
3. Select “Deploy SQL Server big data cluster” from more action drop down.
4. Click on “Deploy SQL Server big data cluster”.
5. Check the check box I accept in the “select the deployment option” screen and click on select button.
6. Select any block and click on next button and again click on next button.
7.Open Voce over and observe the screen reading.
**Actual:**
Voice over does not announce expand and collapse state for docker settings expand and collapse button.
Similar issue has been observed for Service settings screen.
**Expected:**
Voice over should announce the expand and collapse state for docker settings button.
User impact
Screen reader User will not able to know weather the button is expanded or collapse .
**Attachment for reference**
[7940_A11y_AzureDataStudio_ClusterSettings_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3760871/7940_A11y_AzureDataStudio_ClusterSettings_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
this has been fixed by https://github.com/microsoft/azuredatastudio/pull/8452
